index.html

```diff
@@ -72,7 +72,7 @@ <h4>By Dennis Overbye</h4>
                         smoking-gun evidence of inflation, proof, Dr. Kovac and his colleagues say, that Dr. Guth was correct. Inflation has been the workhorse of cosmology for 35 years, though many, including Dr. Guth, wondered whether it could ever
                         be proved.
                         <br><br> If corroborated, Dr. Kovac’s work will stand as a landmark in science comparable to the recent discovery of dark energy pushing the universe apart, or of the Big Bang itself. It would open vast realms of time and space
-                        and energy to science and speculation.
+                        and energy to science.
                         <br><br> Confirming inflation would mean that the universe we see, extending 14 billion light-years in space with its hundreds of billions of galaxies, is only an infinitesimal patch in a larger cosmos whose extent, architecture
                         and fate are unknowable. Moreover, beyond our own universe there might be an endless number of other universes bubbling into frothy eternity, like a pot of pasta water boiling over.
                     </p>
```

Comment:
How the page currently looks when opening it locally(after fixing the image paths):
![Uploading Screenshot_2020-10-23 New York Times Article.png…]()